benchmarks/trace_load_benchmark.py

```diff
@@ -35,8 +45,32 @@ def load_and_parse_trace(
     return pyperf.perf_counter() - t0
 
 
+def load_and_parse_trace_file(loops: int, filename: str, backend: str):
```

Comment:
It would be better to declare backend as an Enum type so that the users and developers would know what the supported backends are. But we can make the change in the next PR.